src/planner/binder/tableref/plan_joinref.cpp

```diff
@@ -77,10 +78,17 @@ void LogicalComparisonJoin::ExtractJoinConditions(
 					continue;
 				}
 			}
-		} else if ((expr->type >= ExpressionType::COMPARE_EQUAL &&
-		            expr->type <= ExpressionType::COMPARE_GREATERTHANOREQUALTO) ||
-		           expr->type == ExpressionType::COMPARE_DISTINCT_FROM ||
-		           expr->type == ExpressionType::COMPARE_NOT_DISTINCT_FROM) {
+		} else if (expr->type == ExpressionType::COMPARE_EQUAL || expr->type == ExpressionType::COMPARE_NOTEQUAL ||
```

Comment:
Unless I'm mistaken this doesn't appear to functionally change anything, it just lists out the enums instead of using a range comparison?

Reply:
Yes, @taniabogatsch and I changed this because we found it more readable this way

Reply:
It was a leftover from the initial debugging, but we decided to keep it for readability, yes.